_includes/doc.tsx

```diff
@@ -168,7 +168,11 @@ export default function Page(props: Lume.Data, helpers: Lume.Helpers) {
           </div>
         </main>
         <aside class="hidden lg:block pb-8 pr-8 col-span-2">
-          <div class="py-2 sticky top-4" id="toc">
+          <div
+            class="py-2 sticky overflow-y-auto top-4 h-[calc(100vh-7rem)]"
+            style={{ scrollbarGutter: "stable", scrollbarWidth: "thin" }}
```

Comment:
Thanks for your contribution!

Would you mind moving the `style={{` as a style for `#toc` at the bottom of styles.css? I want to avoid the in-line styles (other than tailwind classes) if we can.

Reply:
Sure thing!